content/source/docs/enterprise/workspaces/run-ui.html.md

```diff
@@ -33,7 +35,7 @@ By default, run plans require confirmation before TFE will apply them. Users wit
 
 Users can also leave comments if there's something unusual involved in a run.
 
-Note that until you apply or discard a plan, TFE can't start another run in that workspace.
+Note that once the Plan stage is completed, until you apply or discard a plan, TFE can't start another run in that workspace.
```

Comment:
"plan stage" doesn't need a proper noun; it's just a descriptive phrase.

Reply:
Oh yeah, I was going off the [title](https://github.com/hashicorp/terraform-website/blob/8f5d02ed957291c34dc7623506d423a116c3435c/content/source/docs/enterprise/workspaces/run-basics.html.md#L44), but I see the text there also uses lowercase. I'll fix this above too.